.acquia/pipeline.yaml

```
@@ -0,0 +1,36 @@
+---
+type: default
+team: bucc
+group: content-cloud
+service: jvm-threaddump
+
+# The environment container image is used to prepare code versions
+# and tooling for tests during pre, post and build stages.
+environment_image:
+  file: ".acquia/Dockerfile.ci"
+  context: "."
+  args: ""
+  build_args:
+    - secrets:
+      - type: vault
+        key: SSH_KEY
+        value: GIT_SSH_KEY
+        path: secret/pipeline-default/GIT_SSH_KEY
+
+pre_build:
+  code_analysis:
+    required: false
+    project_key: acquia.content-cloud.bucc:jvm-threaddump
+    quality_gate:
+      wait_for_quality_gate: true
+      max_time_limit: 3
+      abort_pipeline: false
+  gradlew assemble:
+    - steps:
+        # - /bin/bash -l -c "./gradlew assemble"
+
+build:
+  service_image:
+    - name: widen/jvm-threaddump
+      file: "Dockerfile"
+      context: "."
```

This should work after https://backlog.acquia.com/browse/DEVOPS-3379 is done.
Adjust pattern and props:

```suggestion
build:
  service_package:
    - type: java 
      pattern: '*.jar'
      props: 
        - maintainer=widen

```